packages/core/src/actions/stop.ts

```diff
@@ -18,7 +18,10 @@ export function stop<
   TContext extends MachineContext,
   TEvent extends EventObject
 >(
-  actorRef: string | Expr<TContext, TEvent, ActorRef<any> | string>
+  actorRef:
+    | string
+    | ActorRef<any>
```

Comment:
Now you can directly specify a `stop(actorRef)`, which is useful in the internal algorithm